engine/src/cmds.h

```diff
@@ -942,6 +942,7 @@ class MCAccept : public MCStatement
 {
 	MCExpression *port;
 	MCExpression *message;
+	MCExpression *name;
```

Comment:
I basically just did that as a way to work out the variant being used in exec but it seems like there's a better plan here with `MCS_name_to_sockaddr`